app/views/journeys/_blank.html.erb

```diff
@@ -0,0 +1,5 @@
+<h2>PRÁZDNY NÁVOD</h2>
```

Comment:
Toto by si zasluzilo lepsi wording, @HanaBene ?

Reply:
Ping @HanaBene

Reply:
uz som ho posielala niekde v tejto issue, je to nizsie. Este iny vstup odomna treba?